core/db/migrate/20170119122701_add_original_return_item_id_to_spree_inventory_units.rb

```diff
@@ -0,0 +1,29 @@
+class AddOriginalReturnItemIdToSpreeInventoryUnits < ActiveRecord::Migration[5.0]
+  def up
+    add_reference :spree_inventory_units, :original_return_item, references: :spree_return_items, index: true
+
+    Spree::InventoryUnit.reset_column_information
+
+    Spree::ReturnItem.where.not(exchange_inventory_unit_id: nil).each do |return_item|
```

Comment:
@priyank-gupta please use `.find_each` instead of each, there could be a huge number of records like this